presto-postgresql/src/main/java/io/prestosql/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -234,11 +234,18 @@ protected ResultSet getTables(Connection connection, Optional<String> schemaName
                 List<JdbcColumnHandle> columns = new ArrayList<>();
                 while (resultSet.next()) {
                     String columnName = resultSet.getString("COLUMN_NAME");
+                    int dataType = resultSet.getInt("DATA_TYPE");
+                    int columnSize = resultSet.getInt("COLUMN_SIZE");
+                    Optional<Integer> decimalDigits = Optional.of(resultSet.getInt("DECIMAL_DIGITS"));
+                    if (dataType == Types.NUMERIC && columnSize == 131089) {
```

Comment:
`columnSize == 131089` is this kind of a magic number for decimal without fixed scale? Is this documented anywhere?

Reply:
`131089` is not documented.
I will use postgres function `format_type(type_oid, typemod)` that results with `numeric` value for such decimals type

Reply:
`131089` comes from here:

Reply:
131089 comes from here:
https://github.com/pgjdbc/pgjdbc/blob/d10ab8d68229b691d9492c942d2cb9183ee9fe32/pgjdbc/src/main/java/org/postgresql/jdbc/TypeInfoCache.java#L793-L796